client/app/hosting/order/hosting-order.service.js

```diff
@@ -3,17 +3,17 @@ angular.module('services').service(
   class HostingOrder {
     /**
      * Constructor
-     * @param Api
+     * @param WucApi
      * @param constants
      */
-    constructor(Api, constants) {
-      this.Api = Api;
+    constructor(WucApi, constants) {
```

Comment:
Are we sure that `constants` will be available from the current module?

Reply:
Good catch

Reply:
Oh, `HostingOrder` is in manager, not in `webUniverseComponents`